exonum/CHANGELOG.md

```diff
@@ -44,6 +44,21 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
   - Remove `ID` constants.
   - Replace `TYPE` constants with a single `SERVICE_ID` constant.
 
+- Several variants were removed from `ApiError` enum.
+
+  Migration path:
+
+  - Use generic `ApiError::BadRequest` variant or create `IronError` directly.
+
+- `CommandExtension` uses `failure::Error` instead of `Box<std::error::Error>`
+  for errors.
+
+  Migration path:
+
+  - `std::error::Error` can be converted to `failure::Error` via `.into()` method.
+
+- `storage::Error` implements `failure::Fail` instead of `std::error::Error`.
```

Comment:
And here?..

Should all the above be somehow combined?..

Reply:
I don't know :-)

On the one hand, they are all part of a single "migrate to failure" PR. On the other hand, each one is a separate user-facing piece of the API.